.eslintrc

```diff
@@ -3,14 +3,18 @@
     "indent": [2, 2],
     "quotes": [2, "single"],
     "linebreak-style": [2, "unix"],
-    "semi": [2, "always"]
+    "semi": [2, "always"],
+    "no-underscore-dangle": 0,
+    "no-param-reassign": 0,
+    "consistent-return": 0,
+    "max-len": 0
```

Comment:
better to use a value that allows for long-ish lines (e.g. 120) rather than disabling the rule entirely